src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -8,6 +8,10 @@
 
 namespace Raven.Client.Documents.Operations.TimeSeries
 {
+    /// <summary>
+    /// Represents a batch operation for time series data, including appends, increments, and deletions, 
+    /// to be performed on a single document’s time series.
+    /// </summary>
     public sealed class TimeSeriesOperation
```

Comment:
let's document the other `public` here.
BTW does the `Name` is mandatory here?

Reply:
done
as yes, `Name` is mandatory - mentioned that in the docs